quelpa.el

```diff
@@ -64,6 +64,12 @@ the `:upgrade' argument."
   :group 'quelpa
   :type 'boolean)
 
+(defcustom quelpa-force-upgrade-p nil
+  "When non-nil, `quelpa-upgrade' will try to upgrade packages to its latest
+version, regardless of the state of your local package repo."
```

Comment:
IIUC, the purpose of this setting is to protect against the case that a user has made local, uncommitted changes in the repo in the Quelpa cache.  If so, maybe the description should mention that, something like:

```
When non-nil, `quelpa-upgrade' will try to upgrade packages to its latest
version, regardless of the state of your local package repo.  Any local, 
uncommitted changes to the repo will be lost.
```

Reply:
Sure, I will update the doc.